### ISSUE TITLE
Light-coloured symbolic icon for power manager (attached).

### ISSUE THREAD

**Issue by Thyname** [shimmerproject/elementary-xfce#158]
I changed the colour to eeeeeeff so it can be used in elementary-xfce-dark panel icons.
[ac-adapter-symbolic.zip](https://github.com/shimmerproject/elementary-xfce/files/3317152/ac-adapter-symbolic.zip)

I don't know if it's useful without the other power manager icons (like battery levels, ups etc.), but posting just in case.

**Comment by Mau-G** [comment]
I'm pretty sure that it already changes the color automatically when using a dark panel
see: https://en.opensuse.org/File:Screenshot_Leap-15.1-Xfce_Desktop.png

**Comment by Thyname** [comment]
Well then there must be a light-coloured icon in your elementary-xfce-dark icon theme. Here in Fedora 30 it was using a hicolor type icon.
I admit I didn't check here upstream before opening this report (haven't checked yet).

Thanks.

**Comment by Mau-G** [comment]
Yes I'm using the latest version from here (git master). Also the color of symbolic icons should be correctly assigned by the gtk theme so that when switching from dark to light you don't need two sets of icons. Maybe try changing you gtk theme or try the version from git here?

**Comment by Thyname** [comment]
Ok, I'm not an expert, but you mean even if there's only a black symbolic icon gtk would be able to re-colourize it?
Because here elementary-xfce-dark/panel/48/ac-adapter.svg ( https://github.com/shimmerproject/elementary-xfce/blob/master/elementary-xfce-dark/panel/48/ac-adapter.svg ) links here https://github.com/shimmerproject/elementary-xfce/blob/master/elementary-xfce/status/48/battery-ac-adapter.svg

I'll check my panel with elementary-xfce and elementary-xfce-dark in a while.

**Comment by Mau-G** [comment]
> Ok, I'm not an expert, but you mean even if there's only a black symbolic icon gtk would be able to re-colourize it?

yes

> Because here elementary-xfce-dark/panel/48/ac-adapter.svg ( https://github.com/shimmerproject/elementary-xfce/blob/master/elementary-xfce-dark/panel/48/ac-adapter.svg ) links here https://github.com/shimmerproject/elementary-xfce/blob/master/elementary-xfce/status/48/battery-ac-adapter.svg

It should read it from here: https://github.com/shimmerproject/elementary-xfce/blob/master/elementary-xfce/devices/symbolic/ac-adapter-symbolic.svg


> 
> I'll check my panel with elementary-xfce and elementary-xfce-dark in a while.

**Comment by ochosi** [comment]
Colored icons are used in the trayicon and notification bubbles, symbolic icons are used in the xfpm panel plugin.

**Comment by Thyname** [comment]
Right, but here it's like this:
elementary icon theme (note black symbolic icon also):
https://pasteboard.co/IkTve9I.png

elementary-xfce (symbolic black icon):
https://pasteboard.co/IkTsKMd.png

elementary-xfce-dark (coloured icon):
https://pasteboard.co/IkTwcG2.png

So unless I create a (light-coloured) symbolic icon in elementary-xfce-dark/panel/22 and 24 it loads this coloured icon.

**Comment by Thyname** [comment]
Read your comment now ochosi, will check the panel plugin.
I'm referring to the tray icon.

**Comment by Mau-G** [comment]
> Read your comment now ochosi, will check the panel plugin.
> I'm referring to the tray icon.

and what about the panel plugin?

**Comment by Thyname** [comment]
Panel plugin works ok, just like yours. So for tray icon, the report is valid and theme needs these light-coloured symbolic icons in panel/22 and panel/24 in elementary-xfce-dark

**Comment by Mau-G** [comment]
> Panel plugin works ok, just like yours. So for tray icon, the report is valid and theme needs these light-coloured symbolic icons in panel/22 and panel/24 in elementary-xfce-dark

Status-notifier plugin has an option to force the use of symbolic icons, you could also try with that. Not sure that the systray aka notification-area has such option. 

@ochosi probably something to bring up at xfce but I've been wondering whether it wouln't be better to make status-notifier and systray use symbolic icons by default so that they are consistent with the panel plugins. Doable?

**Comment by Thyname** [comment]
And as you said, checking now with various icon themes, the icon colour adapts to the gtk theme for the plugin, but not for the tray icon (which needs a light-coloured icon for it, unless it's decided to keep the coloured icon as is).

**Comment by Thyname** [comment]
Just to notice, I'm ok with whatever design decision, especially since the plugin uses the symbolic icon.

What actually triggered this icon issue with me was the fact that current 4.13 xfce-panel is very broken regarding icons when compositing is disabled, particularly in the tray, but also for example in applications menu (which can sometimes not refresh the icons).

I started thinking things were really broken overall in xfce 4.13 regarding icons. But it's basically the panel without compositing. With compositing turned on things appear to work as expected.

**Comment by ochosi** [comment]
The systray currently doesn't support symbolic icons, although I have a branch that tries to implement that (based on what Mate and Cinnamon do). statusnotifier is not part of the default layout yet, but I'll add it

**Comment by newhoa** [comment]
@Thyname is this still an issue? A lot has changed since this issue was raised. systray now supports symbolic icons as an option. elementary-xfce dark themes are gone now, leaving only one theme providing both fullcolor and symbolic icons (no more forced light and dark monochrome-looking color icons).

If it still isn't working right we need to provide the proper symbolic icon (or it may be an issue upstream with systray or the power manager panel plugin).

Thanks!

**Comment by Thyname** [comment]
Hi, I have not been using a computer (except mobile) for a while now, so I don't know and can't evaluate. Thank you. 
@newhoa from what you told, I suppose it should be fine.
